piccolo_api/audit_logs/tables.py

```diff
@@ -0,0 +1,111 @@
+import typing as t
+import uuid
+from datetime import datetime
+from enum import Enum
+
+from piccolo.apps.user.tables import BaseUser
+from piccolo.columns import Text, Timestamp, Varchar
+from piccolo.table import Table
+
+
+class AuditLog(Table):
+    class ActionType(str, Enum):
+        """An enumeration of AuditLog table actions type."""
+
+        CREATING = "CREATING"
+        UPDATING = "UPDATING"
+        DELETING = "DELETING"
+
+    action_time = Timestamp()
+    action_type = Varchar(choices=ActionType)
+    action_user = Varchar()
+    change_message = Text()
```

Comment:
I think one really nice thing would be to be able to which changes were made. There could be a JSON field which would have something like `{'name': 'bob'}` if the name field was changed to 'bob'.

Reply:
Great idea. I will try to implement it.